### PR TITLE
Apply Smart & dumb pattern on Game component

### PR DESCRIPTION
**Pull request by jeymarsonwolox**
Summary

Apply Smart & dumb pattern on Game component

Screenshot

captura de pantalla 2018-11-15 a la s 4 38 50 p m

Trello card

https://trello.com/c/LP12g1LP

### CODE REVIEW

File: tic-tac-toe/src/app/screens/game/index.js [hunk 1]
```diff
@@ -55,15 +39,13 @@ class Game extends Component {
     const history = this.props.history;
```
Comment (julibel) [inline tic-tac-toe/src/app/screens/game/index.js]: lets use destructuring here 😄

**Review by julibel — CHANGES_REQUESTED** [review]
Perfect! great start! lets use this pattern from now on in all future components and screens. Remember.. we use index and layout when the component it's complex and you will need some logic besides the minimum logic for rendering the component. In screens, we almost always have index and layout

**Comment by julibel** [comment]
@jeymarsonwolox one last thing
As component and screens will always be Upper case, we use folders of them with uppercase too.. can you change that?

**Comment by jeymarsonwolox** [comment]
- Use of destructuring structure
- Renaming components folders.

**Review by julibel — APPROVED** [review]
Perfect!

**Pull request merged**